api/services/mailer/templates/alert.js

```diff
@@ -1,11 +1,11 @@
 const layout = require('./layout');
 
 function alert({ errors, reason }) {
-  return layout(`
-    <p><strong>${reason}</strong></p>
-    <p>
+  return layout.render(`
+    <p style="${layout.cssForPs}"><strong>${reason}</strong></p>
```

Comment:
These styles aren't getting added because the function isn't being called (`layout.cssForPs()`)